lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -128,21 +131,33 @@ def make_mock_thread_data(text, thread_id, num_children, group_id=None, group_na
 
 
 def make_mock_request_impl(
-    text,
-    thread_id="dummy_thread_id",
-    group_id=None,
-    commentable_id=None,
-    num_thread_responses=1,
+        course,
+        text,
+        thread_id="dummy_thread_id",
+        group_id=None,
+        commentable_id=None,
+        num_thread_responses=1
```

Comment:
This is just how you should indent function arguments when placing an argument on each line, at least according to pep-8. It helps visually separate the list of parameters from the method body. Without the extra indentation it can be hard to separate the last function argument from the start of the function body:

```python
def my_func(
    arg_1
    arg_2
    ...
    arg_n
):
    body()
```